content/guides/development.md

```diff
@@ -237,8 +238,105 @@ T> Now that your server is working, you might want to give [Hot Module Replaceme
 
 ### Using webpack-dev-middleware
 
-?> Familiar with `webpack-dev-middleware`? We need your help! Please submit a PR to fill in the missing instructions and example here. Make sure to keep it simple as this guide is intended for beginners.
+The `webpack-dev-middleware` provides a way to serve your assets from your express server. The first step is install the webpack and his middleware:
 
+``` bash
+npm install --save-dev webpack webpack-dev-middleware
```

Comment:
No need to install webpack, that's already done, but we do need `express` (if we are to follow your example).